Portfolio: set up work section and about me

jmoseka opened the pull request:
- updated read me
- Implement grid for work section
- Implement flexbox for cards.
- Used ul and li for categories in "About myself" section, languages, Frameworks, and Skills
- Sticked to the figma design (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

MeqdamAlqudah commented on index.html:
```diff
@@ -8,55 +8,335 @@
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </div>
+        <div class="resume-btn">
+          <a href="#" id="resume">Get My Resume</a>
+        </div>
+      </div>
```
- [ ] Please the button here should follow the button styling: Active, Hover and Pressed
![image](https://user-images.githubusercontent.com/74744787/159130876-10e427c7-4f58-475f-bd81-984beadce883.png)

jmoseka replied:
Fixed

MeqdamAlqudah reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @jmoseka,

Good job so far!
✔️ Good README and PR documentation
✔️ Your website is responsive
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jmoseka commented:
Replaced section work with section tag
Replaced card-title with <h2>
Replaced card-post class from div to p
Add enabled, hover, active effects
Replaced section about-me with <section> tag
Used <h2> for about me heading
Used <p> for description in about me section
Used .png to emoji

jmoseka commented:
- Replaced section work with section tag
- Replaced card-title with <h2 tag
- Replaced card-post class from div to p
- Add enabled, hover, active effects
- Replaced section about-me with section> tag
- Used <h2 for about me heading
- Used <p for description in about me section
- Used .png to emoji

jmoseka commented on index.html:
```diff
@@ -8,55 +8,337 @@
+            <li><a href="#" id="ruby">ruby</a></li>
+          </ul>
+        </div>
+        <div class="feature-btn">
```
Resolved. Put section tag - #66 - #217  of index.html file

jmoseka commented on index.html:
```diff
@@ -8,55 +8,335 @@
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-block">
+            <div class="card-title">Professional Art Printing Data</div>
+            <div class="card-post">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
```
Fixed. Put card title line #99 index.html file

jmoseka commented on index.html:
```diff
@@ -8,55 +8,335 @@
+
+        <div class="card">
+          <div class="card-block">
+            <div class="card-title">Professional Art Printing Data</div>
+            <div class="card-post">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </div>
+            <div class="card-tags">
```
Resolved. Put paragram at line #100 index.html file

jmoseka commented on index.html:
```diff
@@ -8,55 +8,335 @@
+      <div class="aboutme-block">
+        <div class="about-me-heading">About me</div>
+        <div class="about-me-desc">
+          Hello I’m a software developer! I can help you build a product ,
```
Resolved. Put section tag for about me #220
<h2 used for the title #222
<p  > for the decription #223

jmoseka commented on index.html:
```diff
@@ -8,55 +8,335 @@
+          <li>Css</li>
+        </ul>
+        <div class="list-title">🟧 Frameworks</div>
+        <ul class="list">
```
Resolved #236 - #281

mmsesay commented on index.html:
```diff
@@ -214,18 +214,18 @@ <h1 class="heading">Hey There. I'm Jamila</h1>
+        </p>
         <div class="resume-btn">
           <a href="#" id="resume">Get My Resume</a>
         </div>
```
- [ ] As suggested by the previous reviewer. The buttons should have some hovering styling changes. I noticed you implemented it on the technologies. However, you need to add the same hovering effect to the buttons including (project cards button, the get resume button).
Please work on this change.

See demo test:|
:------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159134614-7bff7262-fb7d-4197-9372-439c54575cc3.gif)

jmoseka replied:
Added

mmsesay reviewed (changes requested):
Hi @jmoseka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Deepakdanger reviewed (approved):
Hi @jmoseka, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
